### ISSUE TITLE
Scroll with mouse wheel too slow

### ISSUE THREAD

**Issue by klemsFR** [linuxmint/xplayer#30]
Hi,

The scroll performed with the mouse wheel is limited to 1 second steps (forward or backward) which make it totally useless to seek into the media files...

Could you align this on totem behavior (10 or 20 seconds by step?)

Thank you!

**Comment by JosephMcc** [comment]
Seems this was actually changed here to make it lower for short videos: https://github.com/linuxmint/xplayer/commit/918e80811ed4cc442cf7b558547364a5069348a7

**Comment by klemsFR** [comment]
You are right but what the point to have 1 second step for all media shorter than 500 seconds? (many of my mp3 files are shorter than 500seconds!)

I think it would be better to add a minimum value to this scroll step, something like that : 
```
double scroll_step = xplayer->stream_length / 500;
if(scroll_step < MINIMUM_SCROLL_STEP_MS){
double scroll_step=MINIMUM_SCROLL_STEP_MS;
} 
```
Well to me the MINIMUM_SCROLL_STEP_MS should be 10 or 15 seconds and the ` xplayer->stream_length / 500` is to low, for a 2 hours movie it make 14.2 seconds steps.
As for me I would divide it by 100 to make the scroll more efficient.

what so you think?
